webvalve.gemspec

```diff
@@ -16,10 +16,11 @@ Gem::Specification.new do |s|
   s.test_files = Dir["spec/**/*"]
 
   s.add_dependency "rails", ">= 4.2"
-  s.add_dependency "sinatra", "~> 2.0"
-  s.add_dependency "sinatra-contrib", "~> 2.0"
+  s.add_dependency 'sinatra', '>= 1.4', '< 3'
+  s.add_dependency 'sinatra-contrib', '>= 1.4', '< 3'
```

Comment:
Turns out that if you want to support Rails 4.2, you need to stay on sinatra 1.4, due to their shared rack dependency.

Reply:
good catch. i was tracking this down the other day. very annoying.